scout/server/blueprints/institutes/controllers.py

```diff
@@ -422,28 +424,65 @@ def _sort_cases(data, request, all_cases):
     return all_cases
 
 
-def cases(store, request, institute_id):
-    """Preprocess case objects.
+def export_case_samples(institute_id, filtered_cases) -> Response:
+    """Export to CSV file a list of samples from selected cases."""
+    EXPORT_HEADER = [
+        "Sample ID",
+        "Sample Name",
+        "Analysis",
+        "Phenotype",
+        "Sex",
+        "Tissue",
+        "Case Name",
+        "Case ID",
+        "Analysis date",
+        "Case Status",
+        "Track",
+    ]
+    export_lines = []
+    export_lines.append("\t".join(EXPORT_HEADER))  # Use tab-separated values
+    for case in filtered_cases:
+        for individual in case.get("individuals", []):
+            export_line = [
+                individual["individual_id"],
+                individual["display_name"],
+                individual.get("analysis_type").upper(),
+                PHENOTYPE_MAP[individual.get("phenotype", 0)],
+                SEX_MAP[individual.get("sex", 0)],
+                individual.get("tissue_type", "-"),
+                case["display_name"],
+                case["_id"],
+                case["analysis_date"].strftime("%Y-%m-%d %H:%M:%S"),
+                case["status"],
+                case.get("track"),
+            ]
+            export_lines.append("\t".join(str(item) for item in export_line))
+
+    file_content = "\n".join(export_lines)
 
-    Add all the necessary information to display the 'cases' view
+    return Response(
+        file_content,
+        mimetype="text/plain",
+        headers={"Content-Disposition": f"attachment;filename={institute_id}_cases.txt"},
+    )
 
-    Args:
-        store(adapter.MongoAdapter)
-        request(flask.request) request sent by browser to the api_institutes endpoint
-        institute_id(str): An _id of an institute
 
-    Returns:
-        data(dict): includes the cases, how many there are and the limit.
-    """
+def cases(store: MongoAdapter, request: request, institute_id: str) -> dict:
+    """Preprocess case objects for the 'cases' view."""
     data = {}
+
+    # Initialize data (institute info, filters, and case counts)
```

Comment:
Could we add these comments to the docstring as well; they would make a nice little description of what the function does!